[2025-05-14T11:04:07.887+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-14T11:04:07.918+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monitor_T3.monitor_model scheduled__2025-05-14T10:00:00+00:00 [queued]>
[2025-05-14T11:04:07.928+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monitor_T3.monitor_model scheduled__2025-05-14T10:00:00+00:00 [queued]>
[2025-05-14T11:04:07.929+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-05-14T11:04:07.947+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): monitor_model> on 2025-05-14 10:00:00+00:00
[2025-05-14T11:04:07.954+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'monitor_T3', 'monitor_model', 'scheduled__2025-05-14T10:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/monitor_T3.py', '--cfg-path', '/tmp/tmp_lmp2_b7']
[2025-05-14T11:04:07.958+0000] {standard_task_runner.py:91} INFO - Job 13: Subtask monitor_model
[2025-05-14T11:04:07.960+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=185) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-14T11:04:07.961+0000] {standard_task_runner.py:63} INFO - Started process 186 to run task
[2025-05-14T11:04:08.007+0000] {task_command.py:426} INFO - Running <TaskInstance: monitor_T3.monitor_model scheduled__2025-05-14T10:00:00+00:00 [running]> on host 57cf39d9f769
[2025-05-14T11:04:08.102+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monitor_T3' AIRFLOW_CTX_TASK_ID='monitor_model' AIRFLOW_CTX_EXECUTION_DATE='2025-05-14T10:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-14T10:00:00+00:00'
[2025-05-14T11:04:08.103+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-14T11:04:08.213+0000] {logging_mixin.py:188} INFO - [monitor_T3] Prediction: 0.8458118438720703, Ground Truth: 0.65, Error: 0.1958118438720703
[2025-05-14T11:04:08.226+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-14T11:04:08.227+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-14T11:04:08.235+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=monitor_T3, task_id=monitor_model, execution_date=20250514T100000, start_date=20250514T110407, end_date=20250514T110408
[2025-05-14T11:04:08.256+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-14T11:04:08.286+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-14T11:04:08.290+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
